composer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "covergenius/ebay-rest-php-sdk",
+    "description": "https://developer.ebay.com/docs",
+    "type": "library",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "jeff.nielsen",
+            "email": "jeff.n@covergenius.com"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "CoverGenius\\EbayRestPhpSdk\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "CoverGenius\\EbayRestPhpSdk\\Tests\\": "tests/"
+        }
+    },
+    "require": {
+        "php": "^7.3",
+        "guzzlehttp/guzzle": "^6.5",
```

Comment:
Hey Jeff, laravel 8 will require guzzle ^7.0.1, can you please make that the minunm? (Laravel 7 also supports this version).

Reply:
@specialtactics 

I changed this to "guzzlehttp/guzzle": "^6.5|^7.1"

It looks like xpay has some other dependencies that still require `guzzlehttp`. Eg. `prometheus-push-gateway-php-client`

So i'll make a task to get those to a newer `guzzlehttp` version so that xpay isn't bogged down with old versions.

Reply:
Yep, makes sense.